socorro/processor/cache_manager.py

```diff
@@ -333,10 +376,10 @@ def _event_generator(self, nonblocking=False):
 
                             else:
                                 if is_verbose:
-                                    logger.debug("ignored %s %s", path, event)
+                                    logger.debug("unhandled event: %s %s", path, event)
```

Comment:
"ignored" has a specific event meaning, so I changed the msg here to be "unhandled event".